src/pages/Admin/ApplicantSearch/ApplicantList/ListContent/elements.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { FunctionComponent, Fragment, ReactNode } from "react";
+import classNames from "classnames";
+import { IApplicant } from "$interfaces/Applicant";
+import { CareersData } from "$components/CareersDetail/CareersData";
+import { SharedStatusLabel } from "$components/SharedStatusLabel";
+import { APPLICANT_LIST_COLUMNS_VALUES } from "../constants";
+import styles from "./styles.module.scss";
+
+interface IRows {
```

Comment:
se que lo hablamos, pero un arhcivo como bolsa de gatos hacer ruido. es mejor que declare sus componente en carpetas con un archivo index solamente, queda mas prolijo

Reply:
fuertemente de acuerdo. viendo este archivo no entiendo por qué se declararían tantas cosas una abajo de la otra. utils-pattern for sure